packages/import/src/importers/asanaCsv/AsanaCsvImporter.ts

```diff
@@ -1,6 +1,7 @@
 import csv from "csvtojson";
-import j2m from "jira2md";
 import { Importer, ImportResult } from "../../types";
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const j2m = require("jira2md");
```

Comment:
Why does it need to be `require`d?

Reply:
Im honestly not sure. The other package we declare globally works fine :/

I went through a bunch of stack overflow and determined that at least this works... even if it ain't pretty